lib/stitches/auto_fill_test.py

```diff
@@ -0,0 +1,200 @@
+            (18.75, 19.0),
+            (19.75, 19.0),
+            (20.0, 18.0),
+            (20.0, 18.0),
+            (19.5, 18.0),
+            (18.5, 18.0),
+            (17.5, 18.0),
+            (16.5, 18.0),
+            (15.5, 18.0),
+            (14.5, 18.0),
+            (13.5, 18.0),
+            (12.5, 18.0),
+            (11.5, 18.0),
+            (10.5, 18.0),
+            (10.0, 18.0),
+            (10.0, 17.0),
+            (10.0, 17.0),
+            (10.25, 17.0),
+            (11.25, 17.0),
+            (12.25, 17.0),
+            (13.25, 17.0),
+            (14.25, 17.0),
+            (15.25, 17.0),
+            (16.25, 17.0),
+            (17.25, 17.0),
+            (18.25, 17.0),
+            (19.25, 17.0),
+            (20.0, 17.0),
+            (20.0, 16.5),
+            (20.0, 16.0),
+            (20.0, 16.0),
+            (20.0, 16.0),
+            (19.0, 16.0),
+            (18.0, 16.0),
+            (17.0, 16.0),
+            (16.0, 16.0),
+            (15.0, 16.0),
+            (14.0, 16.0),
+            (13.0, 16.0),
+            (12.0, 16.0),
+            (11.0, 16.0),
+            (10.0, 16.0),
+            (10.0, 15.5),
+            (10.0, 15.0),
+            (10.0, 15.0),
+            (10.75, 15.0),
+            (11.75, 15.0),
+            (12.75, 15.0),
+            (13.75, 15.0),
+            (14.75, 15.0),
+            (15.75, 15.0),
+            (16.75, 15.0),
+            (17.75, 15.0),
+            (18.75, 15.0),
+            (19.75, 15.0),
+            (20.0, 14.0),
+            (20.0, 14.0),
+            (19.5, 14.0),
+            (18.5, 14.0),
+            (17.5, 14.0),
+            (16.5, 14.0),
+            (15.5, 14.0),
+            (14.5, 14.0),
+            (13.5, 14.0),
+            (12.5, 14.0),
+            (11.5, 14.0),
+            (10.5, 14.0),
+            (10.0, 14.0),
+            (10.0, 13.5),
+            (10.0, 13.0),
+            (10.0, 13.0),
+            (10.25, 13.0),
+            (11.25, 13.0),
+            (12.25, 13.0),
+            (13.25, 13.0),
+            (14.25, 13.0),
+            (15.25, 13.0),
+            (16.25, 13.0),
+            (17.25, 13.0),
+            (18.25, 13.0),
+            (19.25, 13.0),
+            (20.0, 13.0),
+            (20.0, 12.0),
+            (20.0, 12.0),
+            (20.0, 12.0),
+            (19.0, 12.0),
+            (18.0, 12.0),
+            (17.0, 12.0),
+            (16.0, 12.0),
+            (15.0, 12.0),
+            (14.0, 12.0),
+            (13.0, 12.0),
+            (12.0, 12.0),
+            (11.0, 12.0),
+            (10.0, 12.0),
+            (10.0, 11.5),
+            (10.0, 11.0),
+            (10.0, 11.0),
+            (10.75, 11.0),
+            (11.75, 11.0),
+            (12.75, 11.0),
+            (13.75, 11.0),
+            (14.75, 11.0),
+            (15.75, 11.0),
+            (16.75, 11.0),
+            (17.75, 11.0),
+            (18.75, 11.0),
+            (19.75, 11.0),
+            (20.0, 10.0),
```

Comment:
This final row repeats the first row. I'm assuming this is OK and expected.

Reply:
Again, I'd have to see it visually.  If I had to guess I'd say that one of these is travel and the other is an actual stitched row.  The travel will go around the shape's border because you set underpath to false.

The `tags` attribute on the Stitch objects may help.

Reply:
Yes, the first one is the travel, as confirmed by a second unit test that sets underpath to True in 9b1a06536d734cfacc683ef204eac0e37cb38393